### ISSUE TITLE
Failed notification after finish to render a file

### ISSUE THREAD

**Issue by shakaran** [pitivi/pitivi#146]
I get the following error in terminal:

```
ERROR 11:21:36 freedesktoporgsystem <FreedesktopOrgSystem at 0x7fda0f945750> desktopMessage: desktopMessage: Failed displaying notification: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.Notifications was not provided by any .service files (pitivi/utils/system.py:211)
```

**Comment by aleb** [comment]
What desktop/window manager are you using?

If you run this in a new terminal, does it show a notification?
```
$ python -c "import gi; gi.require_version('Notify', '0.7'); from gi.repository import Notify; Notify.init('App'); Notify.Notification.new('Title', 'Message').show()"
```

**Comment by shakaran** [comment]
@aleb Yes it shows me the notification. I am using Unity

**Comment by aleb** [comment]
And if you run it in `$ flatpak run --command=/bin/bash org.pitivi.Pitivi` ?

**Comment by shakaran** [comment]
$ flatpak run --command=/bin/bash org.pitivi.Pitivi
bash: /usr/local/bin/virtualenvwrapper.sh: No such file or directory

**Comment by aleb** [comment]
After you run `flatpak run --command=/bin/bash org.pitivi.Pitivi` and you get that error, what does `ls /app` print?

**Comment by shakaran** [comment]
$ ls /app
	bin  doc  include  lib  libexec  man  manifest.json  share

**Comment by aleb** [comment]
Fixed by 76e83879028f38f3ab7df8e0aeee37cc7f85d25c
